apps/builder/app/builder/features/settings-panel/variable-popover.tsx

```diff
@@ -560,6 +562,7 @@ export const VariablePopoverTrigger = forwardRef<
                     aria-label="Refresh resource data"
                     prefix={<RefreshIcon />}
                     color="ghost"
+                    disabled={resourcesLoading}
```

Comment:
got confused at first, this would help`disabled={Boolean(resourcesLoading)}`

Reply:
wait is $resourcesLoading a boolean? then it should be something like $isResourceLoading or something